ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/Runner.scala

```diff
@@ -56,7 +57,7 @@ final class Runner[T <: ReadWriteService, Extra](
           // initialize all configured participants
           _ <- Resource.sequence(config.participants.map { participantConfig =>
             val metricRegistry = factory.metricRegistry(participantConfig, config)
-            metricRegistry.registerAll(new JvmMetricSet)
+            metricRegistry.registerAll(JVMServicePrefix, new JvmMetricSet)
```

Comment:
Shouldn't namespacing be the concern of the deployment environment? I.e., in case there are multiple processes running on the same machine the ingested metrics should be namespaced appropriately to avoid clashes. Looking at the broader picture, I'm not sure if generating JVM metrics should be done by our code at all instead of relying on an APM agent to do it for us.

Reply:
> Shouldn't namespacing be a deployment concern? I.e., in case there are multiple processes running on the same machine the ingested metrics should be namespaced appropriately to avoid clashes.

Yes but prefixes simplify the matter (there have been requests in that sense), i.e.  just as we have prefixes for services of the participant, _if_ we publish JVM metrics for it, we should also prefix those.

>  Looking at the broader picture, I'm not sure if generating JVM metrics should be done by our code at all instead of relying e.g. on an APM agent to do it for us.

We basically reuse Dropwizard's intrumentation and it's quite convenient, rather than requiring an external APM agent. But @SamirTalwar-DA and @mziolekda what do you think?

Reply:
We should probably prefix _everything_ with either _daml.participants.{PARTICIPANT_ID}_.

And committer metrics should probably go under their own namespace too.

Reply:
Prefixing all metrics the same specific to a participant server sounds like a good approach!